Dockerfile.ci-portal

```diff
@@ -0,0 +1,17 @@
+ARG REGISTRY
+
+FROM ${REGISTRY}/ubi8/nodejs-16 AS portal-base
+WORKDIR /build
+USER root
+
+FROM portal-base AS portal-build
+COPY /portal/v2 /build/
+RUN npm ci
```

Comment:
I think `npm ci` will be a prerequisite for lint and audit as well, and would need to be moved into a base step. 

EDIT: wait I see this whole build is a prereq for the other two. I don't think that is strictly necessary, lint and audit should run against the code itself and do not require any built artifacts, and the full app build is a generally more expensive/slower step to run than the lint/audit steps will be.

Reply:
We discussed this offline - I've pushed a change that makes the local container build cache the way to optimize this away. In a local env, devs will be able to set `NO_CACHE=false` to get quicker builds.